src/view/com/util/forms/PostDropdownBtnMenuItems.tsx

```diff
@@ -429,20 +436,34 @@ let PostDropdownMenuItems = ({
             </Menu.Item>
           )}
 
+          {!isWeb && (
+            <Menu.Item
+              testID="postDropdownShareBtn"
+              label={_(msg`Share`)}
+              onPress={() => {
+                if (showLoggedOutWarning) {
+                  sharePostLoggedOutWarningPromptControl.open()
+                } else {
+                  onSharePost()
+                }
+              }}>
+              <Menu.ItemText>{_(msg`Share`)}</Menu.ItemText>
+              <Menu.ItemIcon icon={Share} position="right" />
+            </Menu.Item>
+          )}
+
           <Menu.Item
-            testID="postDropdownShareBtn"
-            label={isWeb ? _(msg`Copy link to post`) : _(msg`Share`)}
+            testID="postDropdownCopyLinkBtn"
+            label={_(msg`Copy link to post`)}
             onPress={() => {
               if (showLoggedOutWarning) {
-                loggedOutWarningPromptControl.open()
+                copyPostUrlLoggedOutWarningPromptControl.open()
               } else {
-                onSharePost()
+                onCopyPostUrl()
               }
             }}>
-            <Menu.ItemText>
-              {isWeb ? _(msg`Copy link to post`) : _(msg`Share`)}
-            </Menu.ItemText>
-            <Menu.ItemIcon icon={Share} position="right" />
+            <Menu.ItemText>{_(msg`Copy link to post`)}</Menu.ItemText>
+            <Menu.ItemIcon icon={ChainLink} position="right" />
```

Comment:
question: I chose this icon because it's related to links, but Web used the `Share` icon before. Do we want to keep using the `Share` icon for web?